tests/baselines/reference/jsDeclarationsFunctionClassesCjsExportAssignment.js

```diff
@@ -240,6 +233,13 @@ type Hook = import("./hook");
  * Imports
  */
 type HookHandler = import("./hook").HookHandler;
+/**
+ * Input type definition
+ */
+type Input = {
+    timer: Timer;
+    hook: Hook;
+};

```

Comment:
The order private symbols get added to the output by the symbol tracker alone is potentially slightly different than the one you got when we were using both the symbol tracker and the `includePrivateSymbol` callback being passed around directly. It's just an artifact of the order the functions were getting called in. All the required ones still get added.